engine/reference/builder.md

```diff
@@ -1200,11 +1200,12 @@ into the newly created volume.
 
 ## USER
 
-    USER daemon
+    USER daemon[:daemon]
 
 The `USER` instruction sets the user name or UID to use when running the image
 and for any `RUN`, `CMD` and `ENTRYPOINT` instructions that follow it in the
-`Dockerfile`.
+`Dockerfile`. Optionaly a group name can be set by adding a `:` and a GID, which will be used as the GID 
```

Comment:
This is good info, but let's avoid passive voice:

To specify the group ID (GID), add it as a suffix to the UID, separated by a colon. For instance, `MYUSER:MYGROUP`.

This is just an idea, maybe you can do a better job of conveying it.